analyzer/src/test/kotlin/managers/CarthageTest.kt

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (C) 2020 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.analyzer.managers
+
+import io.kotest.core.spec.style.WordSpec
+import io.kotest.matchers.shouldBe
+
+import io.mockk.every
+import io.mockk.mockkStatic
+
+import java.io.File
+import java.net.URL
+
+import org.ossreviewtoolkit.model.VcsType
+import org.ossreviewtoolkit.utils.test.DEFAULT_ANALYZER_CONFIGURATION
+import org.ossreviewtoolkit.utils.test.DEFAULT_REPOSITORY_CONFIGURATION
+import org.ossreviewtoolkit.utils.test.USER_DIR
+
+class CarthageTest : WordSpec() {
+    private val carthage =
+        Carthage("Carthage", USER_DIR, DEFAULT_ANALYZER_CONFIGURATION, DEFAULT_REPOSITORY_CONFIGURATION)
+
+    init {
+        "resolveDependencies" should {
+            "parse a github dependency" {
+                val cartfile = File("src/test/assets/carthage/Cartfile-github.resolved")
+
+                val result = carthage.resolveDependencies(cartfile).single()
+
+                result.packages.apply {
```

Comment:
We recently [aligned on using with()](https://github.com/oss-review-toolkit/ort/pull/3047) to group test assertion instead of `apply()`.